qiime2/core/archive/provenance.py

```diff
@@ -374,7 +375,9 @@ def reference_plugin(self, plugin):
 
     def capture_env(self):
         return collections.OrderedDict(
-            (d.project_name, d.version) for d in pkg_resources.working_set)
+            (d.metadata["Name"], d.metadata["Version"]) for d in
```

Comment:
Okay @ebolyen I think this works - I compared artifacts from a dev env before/after these changes and the list is the same now.